egs2/tedlium2/asr1/local/download_data.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
I suspect that the data preparation scripts are very similar to `egs`. If that's the case, could you try to use symlink other than a new file, in case to stay sync with the `egs` version

Reply:
> Looks cool! Could you also add an entry in the README of `egs2`?

Sure!

Reply:
> I suspect that the data preparation scripts are very similar to `egs`. If that's the case, could you try to use symlink other than a new file, in case to stay sync with the `egs` version

The join_suffix.py is the same as 'egs', and I think I can use symlink. Other files are different as I have to make some modifications for the 'egs2' style.

Reply:
Cool! Thanks for the clarification.